src/main/java/core/basesyntax/service/impl/CsvFileReaderService.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exceptions.ReadDataFromFileException;
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class CsvFileReaderService implements FileReaderService {
+
+    @Override
+    public String readDataFromFile(String fileName) {
+        File fileFrom = new File(fileName);
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileFrom))) {
+            int value = reader.read();
+            if (value == -1) {
```

Comment:
What does this magic number mean?

Reply:
> What does this magic number mean?

This mean, that file doesn't contain any symbols for reading, this is from logics of read() method, I will do constant for this value.